package.json

```diff
@@ -41,8 +39,6 @@
     "mongoose": "^6.4.4",
     "node-schedule": "^2.1.0",
     "pm2": "^5.2.0",
-    "react-hook-form": "^7.34.0",
-    "react-select": "^5.4.0",
```

Comment:
Have you made sure these can be removed without affecting anyone else's component?